assets/javascripts/discourse/initializers/custom-post-message-callbacks.js

```diff
@@ -77,7 +77,10 @@ export default {
               comments_count: message.comments_count,
             });
 
-            if (post.comments_count - post.comments.length <= 1) {
+            if (
+              post.comments_count - post.comments.length <= 1 &&
+              topicController.currentUser.id !== message.user_id
```

Comment:
This would mean that if I have two tabs open, the second tab will not receive this update. Just curious, is this what core does for the post stream as well? If it is, then I think this solution is fine.